### ISSUE TITLE
Notebook giving 404 permission died

### ISSUE THREAD

**Issue by vsoch** [jupyter/nbviewer#429]
[This notebook](http://nbviewer.ipython.org/github/vsoch/brainmeta/blob/master/image_comparison/notebook/experiment4_residuals.ipynb) was working this morning, but now I am getting a permission denied (403) error.

 In fact, [all of my notebook](http://nbviewer.ipython.org/github/vsoch/brainmeta/blob/master/image_comparison/notebook)s are not working. Any ideas? Something with github?

**Comment by lukeyeager** [comment]
Me too.

It says "400: bad request". Then below it tells me "The error was: HTTP 403: Forbidden". Nice.

**Comment by nigfasa** [comment]
Me uno!

**Comment by tleeuwenburg** [comment]
Yep, mee too! This was working like 5 minutes ago, now everything's failing.

**Comment by tleeuwenburg** [comment]
It just got better for me.

**Comment by vsoch** [comment]
Ditto!

:hatched_chick:

**Comment by bollwyvl** [comment]
Thanks for the report, glad it worked out!

We are aware of intermittent problems with github, related to rate limiting
on the API. We will be working to improve this with more caching in the
future.

One pretty reliable way to get around this type of temporary issue is to
use rawgit
https://rawgit.com/

For the original notebook, you'd get this link:
https://rawgit.com/vsoch/brainmeta/master/image_comparison/notebook/experiment4_residuals.ipynb

which becomes the link:
http://nbviewer.ipython.org/urls/rawgit.com/vsoch/brainmeta/master/image_comparison/notebook/experiment4_residuals.ipynb


Anyhow, if the issue is resolved, please close it: we have a number of
other issues like this open :)

**Comment by tleeuwenburg** [comment]
Is it possible to add some kind of link on the 404 page to the original github URL (org whatever the referred-to resource is) when this occurs so that people can click through themselves?

**Comment by tleeuwenburg** [comment]
Something like:

404: Something went wrong fetching the notebook for rendering. The original URL is <blah> . You may be able to view or retrieve it directly. These issues are often temporary, please check back later.

**Comment by parente** [comment]
#509 has ideas for what to show when a user lands on a 404 page for a non-existent notebook.  @tleeuwenburg has good ideas about what nbviewer could show on other 4xx pages here.
